contracts/EverlongStrategy.sol

```diff
@@ -535,7 +738,7 @@ contract EverlongStrategy is BaseStrategy {
         // Only pre-approved addresses are able to deposit.
         if (_depositors[_depositor] || _depositor == address(this)) {
             // Limit deposits to the maximum long that can be opened in hyperdrive.
-            return IHyperdrive(hyperdrive).calculateMaxLong();
+            return _calculateMaxLong();
```

Comment:
Again, do we really need to be making this change now?

Reply:
+1

Reply:
Actually I think this may include a critical change which is that calculateMaxLong doesn't convert to shares if the shares token is being used (i believe the returned amount is denominated in base).

Happy to include this in another PR though (or not at all and wait til after audit) whatever you think is best

Reply:
We can include it here. I noticed the same thing elsewhere. Sorry for the miscommunication